### PR TITLE
Check autoload on other locations

### PR DESCRIPTION
**Pull request by NicolasDievart**

### CODE REVIEW

File: bin/elao-command-migration [hunk 1]
```diff
@@ -3,6 +3,22 @@
...
 
+function includeIfExists(string $file): bool
+{
+    return file_exists($file) && include $file;
+}
+
+if (
+    !includeIfExists(__DIR__ . '/../../../autoload.php') &&
+    !includeIfExists(__DIR__ . '/../../autoload.php') &&
+    !includeIfExists(__DIR__ . '/../autoload.php') &&
```
Comment (ogizanagi) [inline bin/elao-command-migration]: - `__DIR__ . '/../../../autoload.php'` => `vendor/elao/command-migration/bin/../../../autoload.php` when the package is installed on a project in your vendors
- `__DIR__ . '/../vendor/autoload.php'` => `bin/../vendor/autoload.php` when trying to execute the binary from the checkout repository

I think that's only the two required paths actually

Comment (NicolasDievart) [reply bin/elao-command-migration]: Thanks @ogizanagi, fixed according to your proposal

File: bin/elao-command-migration [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
+if (
+    !includeIfExists(__DIR__ . '/../../../autoload.php') &&
+    !includeIfExists(__DIR__ . '/../vendor/autoload.php')
+) {
+    fwrite(STDERR, 'Install dependencies using Composer.'.PHP_EOL);
+    exit(1);
+}
+
 use Elao\CommandMigration\Command\RunCommand;
 use Symfony\Component\Console\Application;
```
Comment (ogizanagi) [inline bin/elao-command-migration]: `use` directives can still be first, it'll not trigger autoloading :)

File: bin/elao-command-migration [hunk 3]
```diff
@@ -3,6 +3,19 @@
 
 require __DIR__ . '/../vendor/autoload.php';
```
Comment (ogizanagi) [inline bin/elao-command-migration]: Oops. Missed this one! To remove 😄

Comment (NicolasDievart) [reply bin/elao-command-migration]: Fixed

File: bin/elao-command-migration (hunk 2 above)
Comment (NicolasDievart) [reply bin/elao-command-migration]: Moved to the top, thanks

**Pull request merged**